Yosemite/YosemiteTests/Mocks/CardPresentPayments/MockCardReader.swift

```diff
@@ -0,0 +1,23 @@
+@testable import Hardware
+
+struct MockCardReader {
+    static func bbpos() -> CardReader {
```

Comment:
I expect we'll have multiple readers by BBPOS to support as early as next year. Let's call this something like `bbposChipper2XBT` instead